docs/reference/components/sinks/loki.cue

```diff
@@ -116,6 +116,26 @@ components: sinks: loki: {
 				}
 			}
 		}
+		out_of_order_action: {
+			common: false
```

Comment:
I think `common: true` is more fitting here.

Reply:
The `common` attribute is used to indicate those settings that most users would expect to have to set. I'm not sure this would qualify for that.